api/app/hfi/hfi_calc.py

```diff
@@ -17,18 +18,102 @@
                                   required_daily_fields)
 from app.schemas.hfi_calc import (WeatherStationProperties,
                                   FuelType, FireCentre, PlanningArea, WeatherStation)
+from app.fire_behaviour.fuel_types import FUEL_TYPE_DEFAULTS, FuelTypeEnum
 from app.utils.time import get_hour_20_from_date, get_pst_now
+from app.wildfire_one.schema_parsers import WFWXWeatherStation
 from app.wildfire_one.wfwx_api import (get_auth_header, get_stations_by_codes,
                                        get_wfwx_stations_from_station_codes,
                                        get_raw_dailies_in_range_generator)
-from app.wildfire_one.schema_parsers import generate_station_daily
 from app.db.crud.hfi_calc import (get_fire_centre_stations,
                                   get_fire_weather_stations, get_fire_centre_fire_start_ranges,
                                   get_fire_start_lookup)
 
 logger = logging.getLogger(__name__)
 
 
+def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
+                           station: WFWXWeatherStation,
+                           fuel_type: FuelType) -> StationDaily:
+    """ Transform from the raw daily json object returned by wf1, to our daily object.
```

Comment:
This used to be in wfwx, but has all kinds of things that go beyond WFWXWeatherStation slapped on, specific to the hfi response, so I've thrown it in here.